src/components/FormFieldImage.jsx

```diff
@@ -11,24 +12,70 @@ const FormFieldImage = ({
   onFieldChange,
   isRequired,
   style,
-}) => (
-  <>
-    <p className={elementStyles.formLabel}>{title}</p>
-    <input
-      type="text"
-      placeholder={title}
-      value={value}
-      defaultValue={defaultValue}
-      id={id}
-      autoComplete="off"
-      required={isRequired}
-      className={errorMessage ? `${elementStyles.error}` : null}
-      style={style}
-      onChange={onFieldChange}
-    />
-    <span className={elementStyles.error}>{errorMessage}</span>
-  </>
-);
+  inlineButtonText = "Choose Image",
+  siteName
+}) => {
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  const onImageClick = (path) => {
+    setIsModalOpen(false)
+      const event = {
+        target: {
+          id: id,
+          value: path,
+          parentElement: {
+            parentElement: {
+              id: '',
+            },
```

Comment:
Is this `parentElement` being used anywhere?

Reply:
As discussed, this will be handled in a future PR. Filed an issue to address this: https://github.com/isomerpages/isomercms-frontend/issues/177